### DISCUSSION TITLE
What branches / best practices to start development ?

### DISCUSSION THREAD

**Discussion by DzzD** [supermerill/SuperSlicer#2800] [Q&A]
What is the best practice to participate to SuperSlicer, best branche to start developing a new feature ?

Long time ago I've started developing a new feature we have discused about here (adding modes for printer, print, filament) and started from the master, but I am wondering if it was the best choice ?

I let this develoment as it monthes ago as I had to much works elsewhere, but I am considering now to continu working on it, so my question.

**Comment by supermerill** [comment]
The dev branch is where it's going on. This one can be reset and move as needed (to allow to modify commit instead of multiplying them), so don't expect it to be stable (you may need to cherry-pick your commit before submit the pull request)
If you need a more stable start, you can use the latest tag (like the 2.4.58.2), and i'll take care of the squash/cherry pick.
